web/src/components/core/IssuesDrawer.tsx

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright (c) [2024] SUSE LLC
+ *
+ * All Rights Reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the Free
+ * Software Foundation; either version 2 of the License, or (at your option)
+ * any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, contact SUSE LLC.
+ *
+ * To contact SUSE LLC about this file by physical or electronic mail, you may
+ * find current contact information at www.suse.com.
+ */
+
+import React, { forwardRef } from "react";
+import {
+  HelperText,
+  HelperTextItem,
+  NotificationDrawer,
+  NotificationDrawerBody,
+  NotificationDrawerHeader,
+  Stack,
+} from "@patternfly/react-core";
+import Link from "~/components/core/Link";
+import { useAllIssues } from "~/queries/issues";
+import { useInstallerStatus } from "~/queries/status";
+import { IssueSeverity } from "~/types/issues";
+import { InstallationPhase } from "~/types/status";
+import { _ } from "~/i18n";
+
+/**
+ * Drawer for displaying installation issues
+ */
+const IssuesDrawer = forwardRef(({ onClose }: { onClose: () => void }, ref) => {
+  const issues = useAllIssues();
+  const { phase } = useInstallerStatus({ suspense: true });
+  const { issues: issuesByScope } = issues;
+
+  // FIXME: share below headers with navigation menu
+  const scopeHeaders = {
+    users: _("Users"),
+    storage: _("Storage"),
+    software: _("Software"),
+  };
+
+  if (issues.isEmpty || phase === InstallationPhase.Install) return;
+
+  return (
+    <NotificationDrawer ref={ref}>
+      <NotificationDrawerHeader title={_("Preflight checks")} onClose={onClose} />
+      <NotificationDrawerBody className="agama-issues-drawer-body">
+        <Stack hasGutter>
+          <p>
+            {_(
+              "Current installation settings does not meet the necessary requirements for installing the selected product. Click on each section name to access the relevant page and adjust the settings as needed.",
```

Comment:
Reviewers, feel free to improve this text as much as you want. But please resist the temptation to remove it. UI explanations are an aid, not a problem.

Reply:
Regardless of the wording, please fix the broken English:

"Current installation settings **don't** meet the necessary requirements..."

("settings": Plural -> s/"does not"/"do not"/  which nobody ever says (or writes outside of legal documents), so "don't"

Reply:
How about something along the lines of

"You need to make some decisions about some of the settings.
Click on each section name to check and maybe change the setting."

1. Don't make it sound like it's a problem of the installer; it's not.
2. Address the user directly to clarify that it needs user interaction.
3. Keep it simple. Keep not-so-fluent English speakers in mind.
4. Don't fall into the "let's resort to legalese BS" trap:
  First of all, the message needs to be easy to understand, not so much to cover all eventualities.

Reply:
As you said above, I'd like keep some consistency between the user action and the message shown. I mean, 

User clicks on "Install"
The installer shows something that does not install
But the installer also tell the user "Before install, X"

So, user can be sure that this _dialog_ has been for sure shown because as a side effect of clicking on the Install action, not an unexpected message.

Reply:
"Before install" is broken English. It's either "Before installing" or "Before the installation" (or maybe "Before starting the installation".

Reply:
> "Before install" is broken English. It's either "Before installing"

I mean "Before installing", of course. I do not want to break English more that it is already :P

Reply:
Finally, I'll go for "Before installing, you have to make some decisions. Click on each section to review the settings."